zerver/lib/events.py

```diff
@@ -1698,6 +1705,7 @@ def do_events_register(
     user_settings_object = client_capabilities.get("user_settings_object", False)
     linkifier_url_template = client_capabilities.get("linkifier_url_template", False)
     user_list_incomplete = client_capabilities.get("user_list_incomplete", False)
+    include_deactivated_groups = client_capabilities.get("include_deactivated_groups", False)
```

Comment:
Any other suggestions for the name here or we are fine with this?

Reply:
This seems fine, but should be discussed in the API design thread.

Reply:
Posted https://chat.zulip.org/#narrow/stream/378-api-design/topic/deactivated.20groups.20in.20.60GET.60.20endpoints/near/1942417.